src/order/bounded_order.lean

```diff
@@ -1024,6 +1024,20 @@ lemma min_top_right [order_top α] (a : α) : min a ⊤ = a := min_eq_left le_to
 lemma max_bot_left [order_bot α] (a : α) : max (⊥ : α) a = a := max_eq_right bot_le
 lemma max_bot_right [order_bot α] (a : α) : max a ⊥ = a := max_eq_left bot_le
 
+lemma min_bot_left [order_bot α] (a : α) : min ⊥ a = ⊥ := min_eq_left bot_le
+lemma min_bot_right [order_bot α] (a : α) : min a ⊥ = ⊥ := min_eq_right bot_le
+lemma max_top_left [order_top α] (a : α) : max ⊤ a = ⊤ := max_eq_left le_top
+lemma max_top_right [order_top α] (a : α) : max a ⊤ = ⊤ := max_eq_right le_top
```

Comment:
Should these be `simp`?

Reply:
No, `simp` can prove them.

Reply:
How?

Reply:
Both `max_eq_right` and `le_top` a `simp` lemmas (we add `simp` attr to `max_eq_right` in `order/min_max`).

Reply:
[Here](https://github.com/leanprover-community/mathlib/runs/4350601901?check_suite_focus=true) linter complains about simp normal form right before I drop `@[simp]` on these lemmas.